icons/svg-icons.css

```diff
@@ -1,12 +1,7 @@
 /* SVG icons */
```

Comment:
I guess you removed this because `fill: currentColor` is already defined in the `visualizer/style.css`.
Wouldn't it make more sense to leave it here and remove it from style.css?

Reply:
Actually, it's a hangover from the last release. We don't need this CSS definition anymore, because the icons are just using `currentColor` on the fill of the SVG which will be inherited by paths rather than on each path. It's just a case of reducing specificity and making it more future proof should an icon include a `circle` as well as a path etc.

Example here: https://raw.githubusercontent.com/nearform/node-clinic-common/2e54cd7292fa23b5adbd58217f538a626371c99c/assets/arrow-down.svg

Reply:
👍